html-css/week1/nadiia/portfolio/index.html

```diff
@@ -0,0 +1,298 @@
+      <div class="circle b116 a29"></div>
+      <div class="circle b67 a30"></div>
+      <div class="circle b116 a31"></div>
+      <div class="circle b194 a32"></div>
+      <div class="circle b194 a33"></div>
+      <div class="circle b71 a34"></div>
+      <div class="circle b71 a35"></div>
+      <div class="circle b47 a36"></div>
+      <div class="circle b194 a37"></div>
+      <div class="circle b194 a38"></div>
+      <div class="circle b194 a39"></div>
+      <div class="circle b194 a40"></div>
+      <div class="circle b194 a41"></div>
+      <div class="circle b194 a42"></div>
+      <div class="circle b171 a43"></div>
+      <div class="circle b194 a44"></div>
+      <div class="circle b194 a45"></div>
+      <div class="circle b164 a46"></div>
+      <div id="catleft">
+        <div id="cathead">
+          <div class="catear leftear"></div>
+          <div class="catear rightear"></div>
+          <div class="catshead"></div>
+          <div class="catnose"></div>
+          <div class="cateye lefteye"></div>
+          <div class="cateye righteye"></div>
+          <div class="catcheek rightcheek"></div>
+          <div class="catcheek leftcheek"></div>
+        </div>
+        <div id="catbody">
+          <div class="catbreastclose"></div>
+          <div class="catbreast"></div>
+          <div class="cattummyclose"></div>
+          <div class="cattummy"></div>
+          <div class="catpreleg"></div>
+          <div class="catleg"></div>
+          <div class="catpaw leftpaw"></div>
+          <div class="catpaw rightpaw"></div>
+        </div>
+        <div id="cattail">
+          <div class="cattailmiddle"></div>
+          <div class="cattailstart"></div>
+          <div class="cattailclose"></div>
+          <div class="cattailend"></div>
+        </div>
+      </div>
+      <div class="catright">
+        <div class="cathead">
+          <div class="catear leftear"></div>
+          <div class="catear rightear"></div>
+          <div class="catshead"></div>
+          <div class="catnose"></div>
+          <div class="cateye lefteye"></div>
+          <div class="cateye righteye"></div>
+          <div class="catcheek rightcheek"></div>
+          <div class="catcheek leftcheek"></div>
+        </div>
+        <div class="catbody">
+          <div class="catbreast"></div>
+          <div class="cattummy"></div>
+          <div class="catbreastclose"></div>
+          <div class="cattummyclose"></div>
+          <div class="catpreleg"></div>
+          <div class="catleg"></div>
+          <div class="catpaw leftpaw"></div>
+          <div class="catpaw rightpaw"></div>
+        </div>
+        <div class="cattail">
+          <div class="cattailmiddle"></div>
+          <div class="cattailstart"></div>
+          <div class="cattailclose"></div>
+          <div class="cattailend"></div>
+        </div>
+      </div>
+      <div class="welcome">
+        <p class="maintext">Hello, I'm Nadiia</p>
```

Comment:
```suggestion
        <h1 class="maintext">Hello, I'm Nadiia</h1>
```
Headlines help navigate a page better, not just for users with screenreaders, but also for search engine crawlers. It helps them to understand what's the hierarchy or your site, and how to structure it in their algorithms.
This here would most probably be your `<h1>` instead of a paragraph.
You can [read more about the h1-h6 tags at MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) .